sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -4995,12 +4965,538 @@
+                            "conditional": {
+                              "show": "",
+                              "when": null,
+                              "eq": ""
+                            },
+                            "type": "textfield",
+                            "labelPosition": "top",
+                            "inputFormat": "html",
+                            "tags": [],
+                            "properties": {},
+                            "hideLabel": true,
+                            "calculateValue": "value = typeof data.assessment?.totalFederalContribution === 'number' ? '$' + (data.assessment.totalFederalContribution).toLocaleString('en-CA', { minimumFractionDigits: 2 }) : '-';",
+                            "customClass": "font-weight-bold",
+                            "disabled": true
+                          }
+                        ]
+                      },
+                      {
+                        "components": [
+                          {
+                            "autofocus": false,
+                            "input": true,
+                            "tableView": false,
+                            "inputType": "text",
+                            "inputMask": "",
+                            "label": "Text",
+                            "key": "table5Text20",
+                            "placeholder": "",
+                            "prefix": "",
+                            "suffix": "",
+                            "multiple": false,
+                            "defaultValue": "",
+                            "protected": false,
+                            "unique": false,
+                            "persistent": false,
+                            "hidden": false,
+                            "clearOnHide": false,
+                            "spellcheck": false,
+                            "validate": {
+                              "required": false,
+                              "minLength": "",
+                              "maxLength": "",
+                              "pattern": "",
+                              "custom": "",
+                              "customPrivate": false
+                            },
+                            "conditional": {
+                              "show": "",
+                              "when": null,
+                              "eq": ""
+                            },
+                            "type": "textfield",
+                            "labelPosition": "top",
+                            "inputFormat": "html",
+                            "tags": [],
+                            "properties": {},
+                            "hideLabel": true,
+                            "calculateValue": "value = typeof data.assessment?.totalFederalContribution === 'number' ? '$' + (data.assessment.totalFederalContribution).toLocaleString('en-CA', { minimumFractionDigits: 2 }) : '-';",
```

Comment:
This one should be mapped to `totalProvincialContribution` instead `totalFederalContribution`, right?